src/test/java/com/direwolf20/buildinggadgets/common/building/region/RegionTest.java

```diff
@@ -0,0 +1,66 @@
+package com.direwolf20.buildinggadgets.common.building.region;
+
+import com.direwolf20.buildinggadgets.common.building.Region;
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RegionTest {
```

Comment:
Could all test's, which 'modify' the Region, also test the size?

Reply:
Correct me if I'm wrong but shouldn't testing be in a dedicated buildinggadgets.tests package?

Reply:
It should, moment isn't it?

Reply:
I didn't even get that he didn't create a test module!!!

Reply:
Err, moment, they are

Reply:
There not. They're all in common?

Reply:
They are in **src/test** common, not in **src/main** common... Still it should not be in a common package

Reply:
Half right haha!

Reply:
I thought you suppose to put test in the same package but with a different root directory?

Reply:
The common standard is to keep tests out of your main source as you tend to build test separate to the application itself.

Reply:
So basically all test should locate at `src/test/java/com/buildinggadgets/tests` with `com.direwolf20.buildinggadgets.tests` as their package?

Reply:
You can have sub packages to keep order but the tests should be on `com.direwolf20.buildinggadgets.tests` For example `com.direwolf20.buildinggadgets.common.tools.gadgettools` would be `com.direwolf20.buildinggadgets.tools.gadgetToolsTest` if you wanted to do it like that.